tests/security/SingleParticipantWithSecurity/SingleParticipantWithSecurity.mpc

```diff
@@ -0,0 +1,8 @@
+project(SingleParticipantWithSecurity): dcpsexe, dcps_test, dcps_cm, dcps_transports_for_test, opendds_optional_security {
```

Comment:
```suggestion
project: dcps_cm, dcps_rtps_udp, opendds_optional_security {
```

Reply:
Change to `opendds_security` because it requires security and the non-security builds are failing to build this test.